src/css/_main-layout.scss

```diff
@@ -6,19 +6,34 @@
 }
 
 .settings {
-  flex: 0.9 1 0;
+  flex: .9 1 0;
   overflow: hidden;
+  transform: translateY(110%);
```

Comment:
I'm not sure about this. Before this PR, the bits of CSS relating to the "before file open" state are all in one place, since they're relating to the same conceptual state. But the linter has scattered them throughout the file.

Reply:
It's this commit: https://github.com/jakearchibald/svgomg/pull/292/commits/1808d49f98ff3f8cd120b0c3d297975128a316ea

I could revert it, it's just 3 duplicate selectors.

Reply:
Or maybe you could see if you could refactor the code the way you want it without the extra selectors?

Either way I can revert :)

Reply:
Yeah, I think revert that one. The duplicate selectors are deliberate to make things easier to maintain. Happy for them be minified away, but don't think it should happen in the file humans are editing.

Reply:
Cool, I'll revert this patch. Currently we don't have CSS minification, just whitespace cleanup and generally I'm not sure it's worth it, but I have a patch which adds clean-css with sane settings. We can discuss this later.